rolling/distribution.yaml

```diff
@@ -2335,6 +2335,22 @@ repositories:
       url: https://github.com/ethz-asl/libpointmatcher.git
       version: master
     status: maintained
+  librealsense2:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: master
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/IntelRealSense/librealsense2-release.git
```

Comment:
To release into Rolling, we need to have the release repository be in the https://github.com/ros2-gbp organization.  The basic reason is so that we have write access when we go to branch off of Rolling.  To create that, please open a ticket on https://github.com/ros2-gbp/ros2-gbp-github-org

Reply:
Opened PR, waiting for merge:
https://github.com/ros2-gbp/ros2-gbp-github-org/pull/297

Reply:
@clalancette Hi,
We opened a PR 2 weeks ago as you asked.
Can this please be reviewed / merged?